AmazonChimeSDK/AmazonChimeSDKTests/audiovideo/DefaultAudioVideoFacadeTests.swift

```diff
@@ -50,7 +50,7 @@ class DefaultAudioVideoFacadeTests: CommonTestCase {
     }
 
     func testStart_WithConfigArgs() {
-        let audioVideoConfiguration = AudioVideoConfiguration(audioMode: .noAudio, callKitEnabled: true)
+        let audioVideoConfiguration = AudioVideoConfiguration(audioMode: .mono48K, callKitEnabled: true)
```

Comment:
That changes the nature of the test. It was testing `noAudio` and now it tests mono 48K. Is that intentional? What about a test for no audio?

Reply:
Earlier mono was the default value, hence we were testing with a non-default config i.e `noAudio`. Now `stereo48K` is the default, hence using `mono48K` here and keeping `noAudio` as separate tests as we'll probably have to revert Audio checkIn support.